lib/src/screens/home_screen.dart

```diff
@@ -1,10 +1,62 @@
 import 'package:flutter/material.dart';
+import '../../src/constants/constant_colors.dart';
+import '../../src/screens/recipe_screen.dart';
+import '../../src/screens/search_screen.dart';
+import '../../src/screens/user_profile_screen.dart';
 
-class HomeScreen extends StatelessWidget {
+class HomeScreen extends StatefulWidget {
   const HomeScreen({Key? key}) : super(key: key);
 
+  @override
+  State<HomeScreen> createState() => _HomeScreenState();
+}
+
+/// This is the private State class that goes with HomeScreen.
+class _HomeScreenState extends State<HomeScreen> {
+  int _selectedIndex = 1;
+  static const List<Widget> _widgetOptions = <Widget>[
+    SearchScreen(),
+    RecipeScreen(),
+    UserProfileScreen()
+  ];
+  void _onItemTapped(int index) {
+    setState(() {
+      _selectedIndex = index;
+    });
+  }
+
   @override
   Widget build(BuildContext context) {
-    return Scaffold();
+    return Scaffold(
+      body: Center(
+        child: _widgetOptions.elementAt(_selectedIndex),
+      ),
+      bottomNavigationBar: BottomNavigationBar(
+        showSelectedLabels: false,
+        showUnselectedLabels: false,
+        items: <BottomNavigationBarItem>[
+          BottomNavigationBarItem(
+              icon: Image(
+                image: AssetImage('assets/images/search_icon.jpg'),
+                color: _selectedIndex == 0 ? AppColor.green : AppColor.iconText,
+              ),
+              label: 'Search'),
```

Comment:
Can I leave the label blank ?

Reply:
no u can't . Firstly, it is used for widget test easily. Secondly, It shows the meaning of the icon

Reply:
ok i agree